app-devel/mimalloc/spec

```diff
@@ -0,0 +1,4 @@
+VER=2.1.2
+SRCS="tbl::https://github.com/microsoft/mimalloc/archive/refs/tags/v$VER.tar.gz"
```

Comment:
Please use Git source here - GitHub's automatically generated tarballs are not reliable.